test/e2e/cassandracluster_test.go

```diff
@@ -144,3 +150,124 @@ func cassandraClusterRollingRestartDCTest(t *testing.T, f *framework.Framework,
 
 	assert.Equal(t, false, cc.Spec.Topology.DC[0].Rack[0].RollingRestart)
 }
+
+func cassandraClusterInitMultiDCTest(t *testing.T, f *framework.Framework, ctx *framework.TestCtx) {
+	namespace, err := ctx.GetNamespace()
+	if err != nil{
+		t.Fatalf("Could not get namespace: %v", err)
+	}
+
+	cluster := &api.CassandraCluster{
+		TypeMeta: metav1.TypeMeta{
+			Kind:       "CassandraCluster",
+			APIVersion: "db.orange.com/v1alpha1",
+		},
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      "multidc-cluster",
+			Namespace: namespace,
+			Labels:    map[string]string{"cluster": "k8s.pic"},
+		},
+		Spec: api.CassandraClusterSpec{
+			BaseImage: "orangeopensource/cassandra-image",
+			NodesPerRacks: 2,
+			HardAntiAffinity: false,
```

Comment:
You can remove it, it's false by default

Reply:
👍